test/__snapshots__/index.spec.ts.snap

```diff
@@ -6984,6 +7028,32 @@ export abstract class DefaultService {
         });
     }
 
+    /**
+     * @param data Request data
+     * @param options Additional operation options
+     */
+    public serviceWithOneOffRoot(
+        data: {
```

Comment:
Why is the additional object needed here?

Reply:
Hmmm 🤔 it's not needed per se, but it was easier. Let me see if I can improve it actually 👍

Reply:
It's not incorrect.

Reply:
Ok, it's actually needed because `data` there contains both request and query parameters. This was done to have named stuff so it's easier for however uses it to see. We already now of this downside but it actually feels easier to use on the client so leaving it as is 👍

Reply:
@rbruggem ^